pkg/nvi

```diff
@@ -0,0 +1,32 @@
+[mirrors]
+https://sites.google.com/a/bostic.com/keithbostic/files/nvi-1.79.tar.gz
+
+[vars]
+filesize=1295401
+sha512=d1d8adb2aeeee127f5e6fe5cc8b9e3e26fe22084075d17cceecd9cee7956131becee764d721af69626d60d6d75873d5073dc9817995f0274ed307e044921a1d0
+pkgver=1
+
+[deps]
+curses
+
+[build]
+cd build/
+cp -f "$K"/config.sub .
+
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+CPPFLAGS="-D_GNU_SOURCE" CFLAGS="$optcflags" CXXFLAGS="$optcflags" \
+LDFLAGS="$optldflags -Wl,-rpath-link=$butch_root_dir$butch_prefix/lib" \
+  ./configure --prefix="$butch_prefix" --disable-nls $xconfflags \
+	--disable-curses	# use system curses
+
+[ -n "$CROSS_COMPILE" ] && "$K"/libtool.cross.gen --sysroot="$butch_root_dir"
+
+make V=1 -j$MAKE_THREADS
+# make install is broken
+install -Dm755 nvi ${butch_install_dir}${butch_install_prefix}/bin/nvi
+install -Dm755 nex ${butch_install_dir}${butch_install_prefix}/bin/nex
+install -Dm644 $butch_build_dir/docs/USD.doc/vi.man/vi.1 \
```

Comment:
wouldnt $butch_build_dir be equivalento `..` at this point ?

Reply:
yeah, but this line can be called from anywhere, not just the build dir. I can change it, but I don't see what the difference is.

Reply:
yeah, it's just that butch_build_dir is a var that's usually not used in recipes